tfjs-backend-webgl/src/backend_webgl.ts

```diff
@@ -167,9 +166,7 @@ export class MathBackendWebGL extends KernelBackend {
   }
 
   numDataIds() {
-    return this.texData.numDataIds() +
-        (this.cpuBackend ? this.cpuBackend.numDataIds() : 0) -
```

Comment:
Hi Na, I am wondering if it is possible that `cpuBackend.numDataIds()` is not zero if some operators are using cpu. In this case, will `numDataIds()` be correct for webgl? If it doesn't exist, your change looks good for me. Thanks.